test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1169,6 +1171,7 @@ def _revert_pbsconf_server(self, vals_to_set):
                         # start/stop the particular daemon
                         server.pi.initd(server.hostname, initcmd[1],
                                         daemon=initcmd[0])
+                server.isUP()
```

Comment:
On server node along with server we need to wait for other daemons to up and ready depending action.

Reply:
This function can restart multiple daemons, you need to check for each one depending on what's being restarted. 

Also, typo, server.isUP() should be server.isUp()